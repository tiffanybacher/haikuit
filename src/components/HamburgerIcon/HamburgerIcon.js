import React from 'react';

function HamburgerIcon(props) {
  return (
    <button 
      className="Hamburger" 
      onClick={props.toggleMenu}>
      <i className="fas fa-bars"></i>
    </button>
  );
}

export default HamburgerIcon;