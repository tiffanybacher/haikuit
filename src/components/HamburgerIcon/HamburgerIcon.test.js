import React from 'react';
import { shallow } from 'enzyme';
import HamburgerIcon from './HamburgerIcon';

describe('HamburgerIcon', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HamburgerIcon />);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});