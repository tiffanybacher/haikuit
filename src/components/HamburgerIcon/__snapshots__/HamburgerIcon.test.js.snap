// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HamburgerIcon should match snapshot 1`] = `
<button
  className="Hamburger"
>
  <i
    className="fas fa-bars"
  />
</button>
`;
